testing/CMakeLists.txt.orig

```diff
@@ -0,0 +1,50 @@
+cmake_minimum_required(VERSION 3.15)
+
```

Comment:
There's still accidentally commited files left in.

Reply:
Removed